.goreleaser.yml

```diff
@@ -0,0 +1,45 @@
+# This is an example goreleaser.yaml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+before:
```

Comment:
I don't think we need anything in this `before` block since we vendor dependencies and don't generate code